src/rp/index.ts

```diff
@@ -121,6 +127,85 @@ export class RelyingPartySolution {
     );
   }
 
+  /**
+   * Prepare the Verified Presentation token for a received request object in the context of an authorization request flow.
+   * The presentation is prepared by disclosing data from provided credentials, according to requested claims
+   * Each Verified Credential come along with the claims the user accepts to disclose from it.
+   *
+   * The returned token is unsigned (sign should be apply by the caller).
+   *
+   * @todo accept more than a Verified Credential
+   *
+   * @param requestObj The incoming request object, which the requirements for the requested authorization
+   * @param presentation The Verified Credential containing user data along with the list of claims to be disclosed.
+   * @returns The unsigned Verified Presentation token
+   * @throws {ClaimsNotFoundBetweenDislosures} If the Verified Credential does not contain one or more requested claims.
+   *
+   */
+  prepareVpToken(
+    requestObj: RequestObject,
+    [vc, claims]: Presentation // TODO: support multiple presentations
```

Comment:
I would eliminate the array as there is a single vp_token for each presentation

Reply:
It is a single `Presentation`, that's a tuple for the pair {verified_credential, requested_claims}. See https://github.com/pagopa/io-react-native-wallet/blob/8cb654773b055331b2d75352622a97524ca6dd1a/src/rp/types.ts#L77